### PR TITLE
stolostron/backlog#23198--adding content but needs work and explanation

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

**Review by cadawson — APPROVED** [review]
Looks good. Thanks! Merging.

**Comment by cadawson** [comment]
This is already added in the prerequisites section.

File: clusters/hypershift_configure.adoc [hunk 1]
```diff
@@ -15,6 +15,8 @@ true
 [#hypershift-convert-mgt-cluster]
 == Converting an existing cluster to a hosting service cluster
 
+//Chris please add description
+
```
Comment (swopebe) [inline clusters/hypershift_configure.adoc:19]: @cadawson we need to break up headers with a description about why the user needs the section. This is considered a short description and makes is easier to chunk content, also good for accessibility.

**Comment by swopebe** [comment]
This should not have been merged without discussion, clarity, organization

**Pull request closed without merging**